flow/record/tools/rdump.py

```diff
@@ -24,8 +23,9 @@
 
 try:
     # Python 2
-    import urlparse
     from urllib import urlencode
+
+    import urlparse
 except ImportError:
     # Python 3
     import urllib.parse as urlparse
```

Comment:
When touching Python 2 code, just remove it as we don't support it anymore.

Reply:
Code is removed